test/unit/lib/plugins/aws/package/compile/events/cognitoUserPool.test.js

```diff
@@ -339,11 +340,41 @@ describe('AwsCompileCognitoUserPoolEvents', () => {
                 Trigger: 'CustomMessage',
               },
             ],
+            ForceDeploy: undefined,
           },
         });
       });
     });
 
+    it('should create the necessary resources for the most minimal configuration with forceDeploy', async () => {
```

Comment:
Let's rename it to ``should support `forceDeploy` setting``

Reply:
Fixed